docs/Solidity.g4

```diff
@@ -167,6 +171,10 @@ tryStatement : 'try' expression returnParameters? block catchClause+ ;
 // leave this as is, befitting with the Solidity docs.
 catchClause : 'catch' ( identifier? parameterList )? block ;
 
+// ^^FIXME^^: Hey, why is the identifier here meant to be optional? I'd suggest something more like that:
+catchClauseNEW : 'catch' errorTypeName parameterList block ;
+errorTypeName : userDefinedTypeName
```

Comment:
shouldn't it be just an identifier instead of a type name?

Reply:
I first had it set to `identifier`, yes. But I think maybe it should be possible to allow fully qualified identifiers for the error type in the catch block.

Reply:
Ah sorry, maybe we should rename `userDefinedTypeName` to more closely match its purpose.